Hush/Assets/Scripts/Relics/RelicDome.cs

```diff
@@ -0,0 +1,16 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using Common;
+
+public class RelicDome : MonoBehaviour
+{
+    public int keysNeededToUnlock = 0;
+
+    void OnTriggerEnter(Collider collider) {
+        if (collider.gameObject.tag == Tags.Player && GameMaster.NumOfKeys == keysNeededToUnlock) {
```

Comment:
to stay consistent with Jun's previous comments, you could use the CompareTags function we have:
```suggestion
        if (collider.gameObject.CompareTag(Tags.Player) && GameMaster.NumOfKeys == keysNeededToUnlock) {
```